src/components/help-section.tsx

```
@@ -1,73 +1,24 @@
-import { MinimalCard } from './cards/minimal-card'
-import { MinimalCardDescription } from './cards/minimal-card-description'
-import { MinimalCardImage } from './cards/minimal-card-image'
-import { MinimalCardTitle } from './cards/minimal-card-title'
+import type { HelpSection } from '@/types';
+import { MinimalCard } from './cards/minimal-card';
+import { MinimalCardDescription } from './cards/minimal-card-description';
+import { MinimalCardImage } from './cards/minimal-card-image';
+import { MinimalCardTitle } from './cards/minimal-card-title';
 
-export default function HelpSection() {
+export default function HelpSection({ data }: { data: HelpSection }) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'HelpSection'. Consider to delete it or rename it.

'HelpSection' is defined here:


(lint/suspicious/noRedeclare)